docs/verifiable-document/document-template.md

```diff
@@ -215,14 +219,14 @@ On the build page, enter `npm run build` as the "Build command" and `dist` as th
 
 ![Sample Deployed URL](/docs/verifiable-document/document-template/netlify-deployed.png)
 
-Once the site has been deployed, you will obtain the url to the document renderer site. In the above example, the url is `https://frosty-joliot-c02c3d.netlify.com/`.
+Once the site has been deployed, you will obtain the URL to the document renderer site. In the above example, the URL is `https://frosty-joliot-c02c3d.netlify.com/`.
 
-Note that the website will be an empty page when viewed directly.
+Note that the website will be an empty page when viewed directly. This is normal because it is not meant to be viewed directly through a web browser.
```

Comment:
mayne in bold ? **This is normal because it is not meant to be viewed directly through a web browser.**